src/animations/Shine.tsx

```diff
@@ -26,13 +26,16 @@ export class Shine extends React.Component<IShine> {
   public render() {
     const { children, style } = this.props;
 
-    const left = this.animation.interpolate({
+    const translateX = this.animation.interpolate({
       inputRange: [START_VALUE, END_VALUE],
-      outputRange: ["0%", "100%"],
+      outputRange: ["0%", "250%"],
```

Comment:
May I ask why 250%? 😊

Reply:
Sure. The width of the animated view is 40% of the parent. In order to Translate over the whole surface of the parent, we need to Translate with 2.5 of itself meaning 100% of the parent

Reply:
I'm seeing the following error: https://github.com/facebook/react-native/blob/aee88b6843cea63d6aa0b5879ad6ef9da4701846/ReactAndroid/src/main/java/com/facebook/react/bridge/ReadableNativeMap.java#L132

![Screenshot_20200918-082052](https://user-images.githubusercontent.com/3874873/93563395-15830d00-f988-11ea-8f00-38b3589c03ab.png)

Looks like % are not valid as a transform value. See the API here: https://reactnative.dev/docs/transforms#transform

Thank you for the effort on this pull requests 😊 🙏

Reply:
I see, this is on Android, right? Hmmm... To be fair, I only tested IOS, which was working. I will look into it, when I have some more time 👍

Reply:
Yep that's on android :)